apis/achievements.py

```diff
@@ -0,0 +1,44 @@
+from flask import request, Blueprint, session
+from util.decorators import check_user
+
+import db
+from db import get_db
+from pymysql.cursors import DictCursor
+
+from wikispeedruns import achievements
+
+achievements_api = Blueprint('achievements', __name__, url_prefix='/api/achievements')
+
+
+@achievements_api.get('/get/<int:run_id>')
+def get_new(run_id):
```

Comment:
I would recommend a more descriptive name here for the endpoint. `get` is a bit generic, also idk if it should be a get because the response would change after multiple runs. Maybe it should be a PATCH with a name related to `process`